packages/suite/src/utils/wallet/__fixtures__/coinjoinUtils.ts

```diff
@@ -2,6 +2,79 @@ import {
     DEFAULT_TARGET_ANONYMITY,
     ESTIMATED_MIN_ROUNDS_NEEDED,
 } from '../../../services/coinjoin/config';
+import * as coinjoinUtils from '../coinjoinUtils';
+
+const baseUtxo = {
+    txid: '1',
+    vout: 1,
+    amount: '100',
+    blockHeight: 100,
+    path: 'string',
+    confirmations: 100,
+};
+
+export const breakdownParams: Parameters<typeof coinjoinUtils.breakdownCoinjoinBalance>[0] = {
+    targetAnonymity: 80,
+    anonymitySet: {
+        one: 1,
+        two: 100,
+        three: 30,
+    },
+    utxos: [
+        {
+            address: 'one',
+            ...baseUtxo,
+        },
+        {
+            address: 'two',
+            ...baseUtxo,
+        },
+        {
+            address: 'three',
+            ...baseUtxo,
+        },
+    ],
+};
+
+export const calculateProgressParams: Parameters<
```

Comment:
would be nice to add more critical cases like no utxos, just one utxo, everything above targetAnonymity...

Reply:
f56eaa083acc69a8652848b738da50547c3beaa6